doc/sphinx/introduction.rst

```diff
@@ -30,33 +30,30 @@ Coq has an interactive mode in which commands are interpreted as the
 user types them in from the keyboard and a compiled mode where commands
 are processed from a file.
 
--  The interactive mode may be used as a debugging mode in which the
-   user can develop his theories and proofs step by step, backtracking
-   if needed and so on. The interactive mode is run with the `coqtop` 
-   command from the operating system (which we shall assume to be some
-   variety of UNIX in the rest of this document).
+-  The interactive mode may be used as a debugging mode in which the users can
+   develop their theories and proofs step by step, backtracking if needed and
+   so on. The interactive mode is generally run with the help of an IDE, such
```

Comment:
I'd also remove IDE, since nor CoqIDE nor PG are IDEs.
You could use "Text Editor, either devoted to Coq as CoqIDE or general purpose as Emacs using the PG package"

Reply:
I think they are IDEs, rudimentary IDEs but IDEs nonetheless, since they embed a text editor + a way of communicating back and forth with Coq.